packages/test-runner-puppeteer/src/index.ts

```diff
@@ -1,2 +1,2 @@
 export { puppeteerLauncher } from './puppeteerLauncher.js';
-export { devices } from 'puppeteer';
+export { KnownDevices } from 'puppeteer';
```

Comment:
this is kind a breaking change? as such good to document in the migration guide?

but if we do it anyway, I'd rather remove this export, as people should import this directly from `puppeteer`, wdyt?

Reply:
Good question. Thinking about it, I do not see a clear advantage of reexporting it, so I agree with you that letting people import it from puppeteer directly would be the better option. But this will be a breaking change I guess, so how should I approach it?

Reply:
@bashmish; any input here?

Reply:
actually `@web/test-runner-puppeteer` is on 0.x, so keep it `minor` in the changeset, since in SemVer resolution it will behave like major update
the outcome should be a bump from 0.15.0 to 0.16.0
mention this change in the MIGRATION.md

Reply:
@bashmish I removed the KnownDevices export, created the changeset, and added a MIGRATION.md. I could not find an existing MIGRATION.md, so I created a new one and kept it as simple as possible.

Reply:
In addition, I had to re-create the package-lock.json again, because it had conflicts with master. Could you double check it is still ok?